qiling/os/windows/dlls/msvcrt.py

```diff
@@ -185,43 +185,6 @@ def hook__get_initial_narrow_environment(ql, address, params):
     return ret
 
 
-def printf(ql, address, fmt, params_addr, name):
```

Comment:
you might wanna keep this part for if ql.shellcoder == True and normal Windows goes to your version

Reply:
I don't understand why, the function was moved to utils since, it should not create a problem.

Reply:
i think quynh move it. for what reason i got no idea. maybe @aquynh can answer